platform-sdk/platform-apps/demos/CryptocurrencyDemo/src/main/java/com/swirlds/demo/crypto/CryptocurrencyDemoState.java

```diff
@@ -211,9 +211,7 @@ public synchronized CryptocurrencyDemoState copy() {
     public void handleConsensusRound(
             @NonNull final Round round,
             @NonNull final PlatformStateModifier platformState,
-            @NonNull
-                    final Consumer<List<ScopedSystemTransaction<StateSignatureTransaction>>>
-                            stateSignatureTransactions) {
+            @NonNull final Consumer<ScopedSystemTransaction<StateSignatureTransaction>> stateSignatureTransactions) {
```

Comment:
Probably it makes more sense to have single wording in all places, since now we consume a single transaction at a time.

stateSignatureTransactions -> stateSignatureTransaction